docs/sources/mimir/configure/about-ingester-adaptive-limiters.md

```diff
@@ -0,0 +1,60 @@
+---
+aliases:
+  - ../configuring/configuring-ingester-adaptive-limiters/
+description: Learn about ingester adaptive limiters.
+menuTitle: Ingester adaptive limiters
+title: About Grafana Mimir ingester adaptive limiters
+---
+
+# About Grafana Mimir ingester adaptive limiters
+
+Mimir's adaptive concurrency limiters can be used to guard against ingester overload, by automatically adapting concurreny limits based on indications of overload.
+
+## How do adaptive limiters work?
+
+Adaptive limiters detect overload by observing response times, inflight requests, and throughput.
+
+When recent response times increase significantly relative to the longer term trend, an adaptive limiter will temporarily decrease concurrency limits to avoid potential overload. Similarly, when increasing inflight requests correlate with flat or decreasing throughput and increasing response times, this is taken as a sign of overload and the concurrency limit is decreased.
+
+## How do adaptive limiters behave in practice?
```

Comment:
Can we simplify this heading to `How adaptive limiters work` ?

Reply:
The section above was meant to be about how they work, and this section was meant to be more about how they behave. Did you want to shorten this section to `How do adaptive limiters behave?`

Reply:
I think we can actually do away with the heading and just have everything under `How adaptive limiters work`.